wordpress-mobile/WordPress-Android
# Add error handling to the account closure flow

Fixes #18406

### Description

This PR adds error handling to the account closure flow. It depends on FluxC changes brought by this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/18407.

### To test:

Please test using [the steps in the feature branch PR description](https://github.com/wordpress-mobile/WordPress-Android/pull/18412).


PR submission checklist:

See feature branch PR for checklist.
